setup/bootstrap.sh

```diff
@@ -2,18 +2,36 @@
 #########################################################
 # This script is intended to be run like this:
 #
-#   curl https://mailinabox.email/setup.sh | sudo bash
+#   wget https://mailinabox.email/setup.sh -qO - | sudo bash -s
+#   curl -s https://mailinabox.email/setup.sh | sudo bash -s
 #
 #########################################################
 
 if [ -z "$TAG" ]; then
 	TAG=v0.29
 fi
 
+if [[ "$#" -ne 0 ]]; then
+	echo "Usage: \"wget https://mailinabox.email/setup.sh -qO - | sudo bash -s\" or \"curl -s https://mailinabox.email/setup.sh | sudo bash -s\"" >&2
+	exit 1
+fi
+
 # Are we running as root?
 if [[ $EUID -ne 0 ]]; then
-	echo "This script must be run as root. Did you leave out sudo?"
-	exit
+	echo "This script must be run as root. Did you leave out sudo?" >&2
+	exit 1
+fi
+
+# Check if on Linux
+if ! echo "$OSTYPE" | grep -iq "linux"; then
+	echo "Error: This script must be run on Linux." >&2
+	exit 1
+fi
+
+# Check connectivity
+if ! ping -q -c 3 mailinabox.email > /dev/null 2>&1; then
```

Comment:
The reliance on mailinabox.email is https, not ping. @JoshData, I don't suppose you have a health or liveness endpoint? Instead of pinging, you could do something such as `curl https://mailinabox.email/health | grep OK`

Reply:
To clarify, given a choice between ping and nothing, your adding ping here is infinitely better. ;)